bundles/org.openhab.binding.mqtt.espmilighthub/src/main/resources/OH-INF/i18n/mqtt.properties

```diff
@@ -1,3 +1,20 @@
+# thing types
+
+thing-type.mqtt.cct.label = Milight CCT
```

Comment:
I'm a bit confused about this line in this module but if that is fine LGTM

Reply:
The MQTT Binding also has a bundle for integrating an open source based Milight hub using MQTT (see [add-on docs](https://www.openhab.org/addons/bindings/mqtt.espmilighthub/#espmilighthub-binding)).

This is what all MQTT Binding things look like in the UI :slightly_smiling_face: 


![Screenshot from 2022-02-20 16-55-32](https://user-images.githubusercontent.com/12213581/154851722-cfe289ed-a9cd-4765-ba85-ff4c947a9d7e.png)